<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimerListPanel</class>
 <widget class="QFrame" name="PrimerListPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="actionPanel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="addPrimers">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>8</y>
        <width>26</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>26</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Add new primers</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resources/AlignShop.qrc">
        <normaloff>:/icons/16x16/actions/add.png</normaloff>:/icons/16x16/actions/add.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="primersTitle">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Primer Pairs</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="showInfo">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>8</y>
        <width>26</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>26</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cabin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Show param info</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resources/AlignShop.qrc">
        <normaloff>:/icons/16x16/actions/show_info.png</normaloff>:/icons/16x16/actions/show_info.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="contentPanel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableView" name="primersGrid">
        <property name="font">
         <font>
          <family>Cabin</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="zeroPrimersLabel">
        <property name="font">
         <font>
          <family>Cabin</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>No primers have been created for this sequence.  Click the plus icon (above) to add a new primer.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/AlignShop.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addPrimers</sender>
   <signal>clicked()</signal>
   <receiver>PrimerListPanel</receiver>
   <slot>onAddPrimerClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showInfo</sender>
   <signal>clicked()</signal>
   <receiver>PrimerListPanel</receiver>
   <slot>onShowInfoClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddPrimerClicked()</slot>
  <slot>onShowInfoClicked()</slot>
 </slots>
</ui>
